backend/note_taking/serializers/block_list_serializer.py

```
@@ -65,6 +106,19 @@ def create(self, validated_data):
 
+                    for idx, following_block in enumerate(following_blocks):
+                        offset = idx + 1
+
+                        following_block["position"] = item["position"] + offset
+                        
+                        try:
+                            following_block = Block.objects.create(**following_block)
+                            
+                            blocks.append(following_block)
+                            temp_id_mapping[following_block.id] = following_block.id
+                        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)